sleap/io/cameras.py

```diff
@@ -853,13 +864,62 @@ def update_points(
             exclude_complete: If True, then do not update points that are marked as
                 complete. Default is True.
         """
+        # Ensure we are working with a float array
+        points_3d = points_3d.astype(float)
+
+        # Check if points are 3D
+        is_3d = points_3d.shape[-1] == 3
+        if not is_3d:
+            raise ValueError("Expected 3D points with shape (M, T, N, 3).")
+        
+        # Check that the correct shape was passed in
+        n_views, n_instances, n_nodes, n_coords = points_3d.shape
+        assert n_views == len(
+            self.cams_to_include
+        ), f"Expected {len(self.cams_to_include)} views, got {n_views}."
+        assert n_instances == len(
+            instance_groups
+        ), f"Expected {len(instance_groups)} instances, got {n_instances}."
+        assert n_coords == 3, f"Expected 3 coordinates, got {n_coords}."
+
+        # Reproject 3D points into 2D points for each camera view
+        pts_reprojected = reproject(
+            points_3d,
+            calib=self.session.camera_cluster,
+            excluded_views=self.excluded_views,
```

Comment:
Similarly, there is no `InstanceGroup.excluded_views`. I think we should either add this as an input into `InstanceGroup.update_points` (and get rid of the `cams_to_include` input)  or derive it from the existing `cams_to_include` input and `InstanceGroup.camera_cluster` attribute.